api/src/main/java/org/pmiops/workbench/db/dao/UserRecentResourceServiceImpl.java

```diff
@@ -104,6 +110,28 @@ public void updateConceptSetEntry(long workspaceId, long userId, long conceptSet
         String.valueOf(conceptSetId));
   }
 
+  @Override
+  public void updateDataSetEntry(long workspaceId, long userId, long dataSetId) {
+    // Save entry for data set in the new user_recently_modified table only when the feature flag is on
+    // this is because workbench doesnt store dataset information in the existing table user_recent_resources,
+    if (workbenchConfigProvider.get().featureFlags.enableDSCREntryInRecentModified) {
+      updateUserRecentlyModifiedResourceEntry(
+          workspaceId,
+          userId,
+          DbUserRecentlyModifiedResource.DbUserRecentlyModifiedResourceType.DATA_SET,
+          String.valueOf(dataSetId));
+    }
+  }
+
+  @Override
+  public void updateCohortReviewEntry(long workspaceId, long userId, long cohortReviewId) {
+    updateUserRecentlyModifiedResourceEntry(
```

Comment:
Not being used right now, will enclose with feature flag when i'll be doing inserts

Reply:
Nvm i updated it